### PR TITLE
add tool to check memory of different objects

### PR DESCRIPTION
**Pull request by mikedoan**
add tool to check memory of different objects.
Will add more main objects after this.

### CODE REVIEW

File: internal/configs/node/config.go [hunk 1]
```diff
@@ -69,6 +69,8 @@ type ConfigType struct {
 	isBeacon bool        // whether this node is beacon node doing consensus or not
 	ShardID  uint32      // ShardID of this node
 	role     Role        // Role of the node
+	Port     string
```
Comment (rlan35) [inline internal/configs/node/config.go]: Add description for ip/port.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
Jenkins, please test this.

**Comment by mikedoan** [comment]
Jenkins, please test this.

**Comment by LeoHChen** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
Jenkins, test this please.

**Comment by mikedoan** [comment]
```
./test/deploy.sh: line 24: 35636 Killed: 9               $DRYRUN "${ROOT}/bin/harmony" "${args[@]}" 2>&1
     35637 Done                    | tee -a "${LOG_FILE}"
./test/deploy.sh: line 24: 35638 Killed: 9               $DRYRUN "${ROOT}/bin/harmony" "${args[@]}" 2>&1
     35639 Done                    | tee -a "${LOG_FILE}"
+ check_result
+ find tmp_log/log-20190604-125011 -name 'leader-*.log'
+ find tmp_log/log-20190604-125011 -name 'validator-*.log'
+ find tmp_log/log-20190604-125011 -name 'archival-*.log'
+ echo ====== RESULTS ======
====== RESULTS ======
++ ./test/../test/cal_tps.sh tmp_log/log-20190604-125011/all-leaders.txt tmp_log/log-20190604-125011/all-validators.txt
expr: not a decimal number: '"9000"'
+ results='1 shards, 49 consensus, 66 total TPS, 9 nodes, 10 total nodes, total signatures
127.0.0.1, 49, 66.2486
peerID: 9002, numOfConsensus: 50
peerID: 9003, numOfConsensus: 50
peerID: 9001, numOfConsensus: 50
peerID: 9004, numOfConsensus: 50
peerID: 9005, numOfConsensus: 50
peerID: 9007, numOfConsensus: 50
peerID: 9006, numOfConsensus: 50
peerID: 9008, numOfConsensus: 50
peerID: 9009, numOfConsensus: 50'
+ echo 1 shards, 49 consensus, 66 total TPS, 9 nodes, 10 total nodes, total signatures 127.0.0.1, 49, 66.2486 peerID: 9002, numOfConsensus: 50 peerID: 9003, numOfConsensus: 50 peerID: 9001, numOfConsensus: 50 peerID: 9004, numOfConsensus: 50 peerID: 9005, numOfConsensus: 50 peerID: 9007, numOfConsensus: 50 peerID: 9006, numOfConsensus: 50 peerID: 9008, numOfConsensus: 50 peerID: 9009, numOfConsensus: 50
+ tee -a tmp_log/log-20190604-125011/r.log
1 shards, 49 consensus, 66 total TPS, 9 nodes, 10 total nodes, total signatures 127.0.0.1, 49, 66.2486 peerID: 9002, numOfConsensus: 50 peerID: 9003, numOfConsensus: 50 peerID: 9001, numOfConsensus: 50 peerID: 9004, numOfConsensus: 50 peerID: 9005, numOfConsensus: 50 peerID: 9007, numOfConsensus: 50 peerID: 9006, numOfConsensus: 50 peerID: 9008, numOfConsensus: 50 peerID: 9009, numOfConsensus: 50
+ echo 1 shards, 49 consensus, 66 total TPS, 9 nodes, 10 total nodes, total signatures 127.0.0.1, 49, 66.2486 peerID: 9002, numOfConsensus: 50 peerID: 9003, numOfConsensus: 50 peerID: 9001, numOfConsensus: 50 peerID: 9004, numOfConsensus: 50 peerID: 9005, numOfConsensus: 50 peerID: 9007, numOfConsensus: 50 peerID: 9006, numOfConsensus: 50 peerID: 9008, numOfConsensus: 50 peerID: 9009, numOfConsensus: 50
```

**Comment by mikedoan** [comment]
Submitting the code now.

**Pull request merged**